src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml

```diff
@@ -1,69 +1,56 @@
-admin_metas_index:
-    path: /
-    methods: GET
-    defaults:
-        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:index'
-        _legacy_controller: AdminMeta
-        _legacy_link: AdminMeta
+admin_meta_index:
```

Comment:
BC Break ⚠️ 
Unfortunately this page has been released in 1.7.5.0 *and* displayed (which is not the case of webservice keys page for example). So we need to make sure it is retro-compatible.

That being said 🤔 I checked and I discovered there is no alias feature for routes ! For SF services you can alias them, but it's not possible for routes !

And I'm not even sure we can just copy-paste as there will be pathes overlap ...

Aaaaargh 😅 
@eternoendless @mickaelandrieu @sarjon any ideas or tricks ?

Reply:
No, you can rename the route but we must make sure that the old route name is still usable.

Reply:
Looks like you can just preserve the old route and add the new one (thanks @mickaelandrieu)
https://gist.github.com/mickaelandrieu/537d7b146d14e3eb271b4dfbce071639

Reply:
got some errors when preserving both routes - I just renamed back to the previous name due to I think it looks good enough `admin_metas_index`.

Reply:
Issue to be solved in https://github.com/PrestaShop/PrestaShop/issues/12432